test/src/unit-capi-attributes.cc

```diff
@@ -286,8 +286,11 @@ TEST_CASE_METHOD(
 
 TEST_CASE_METHOD(
     Attributesfx,
-    "C API: Test attributes with tiledb_blob datatype",
-    "[capi][attributes][tiledb_blob]") {
+    "C API: Test attributes with std::byte",
+    "[capi][attributes][byte]") {
+  auto datatype = GENERATE(TILEDB_BLOB, TILEDB_GEOM_WKB, TILEDB_GEOM_WKT);
```

Comment:
Do we have tests to validate our C++ api changes for TILEDB_GEOM_WKB and TILEDB_GEOM_WKT?